README.md

```diff
@@ -1 +1,41 @@
-# docs-website
\ No newline at end of file
+# [Trellis Toolkit - Main Docs Website](https://github.com/trellistk/docs-website)
+
+<div align="center">
+
+[![Watchers](https://img.shields.io/github/watchers/trellistk/docs-website?style=social)](https://github.com/trellistk/docs-website) [![Activity](https://img.shields.io/github/commit-activity/m/trellistk/docs-website)](https://github.com/trellistk/docs-website/pulse)
+[![Issues Open](https://img.shields.io/github/issues/trellistk/docs-website?logo=readthedocs&logoColor=green)](https://github.com/trellistk/docs-website/issues) [![Issues Closed](https://img.shields.io/github/issues-closed/trellistk/docs-website?logo=readthedocs&logoColor=yellow)](https://github.com/trellistk/docs-website/issues?q=is%3Aissue+is%3Aclosed)
+</div>
+
+## Table of Contents
+
+- [Overview]('#overview')
+- [Installation](#installation)
+- [Contribute](#contribute)
+- [Contributors](#contributors)
+- [License](#license)
+
+### Overview
+
+Overview of docs website and what it is currently about.
+
+### Installation
+
+Instructions for setting up code to run Docs-Website.
+
+Steps
+
+- Git Repo Clone
+- Serverless Framework Setup
+- Dev Environment Setup
```

Comment:
Can you add terminal commands for how to install dependencies?

Also how to start and view the site locally.